src/main/java/gui/headerarea/ToolView.java

```diff
@@ -15,6 +17,12 @@
 
     private List<ToolButton> buttonList;
 
+    private String style = "-fx-background-color: "
```

Comment:
Can't couple that well between java and css, sadly. This means I'll lose a lot of the tweakability changing variables/properties gives me. :(